docs/pages/tutorial/build-a-screen.mdx

```diff
@@ -377,7 +380,7 @@ const styles = StyleSheet.create({
 });
 ```
 
-Let's take a look at our app on iOS, Android and the web:
+Let's take a look at our app on Android, iOS and the web:
```

Comment:
One small suggestion to revert this change is because if you look at the actual screenshot (or video in some other but similar cases) it refers to the actual platforms shown in that order.

TLDR: it was intentional.